docs/content/reference/data_types/tensor.md

```diff
@@ -2,9 +2,29 @@
 title: Tensor
 order: 23
 ---
+
+Tensors are N-dimensional data matrix with homogeneous type. Supported types are:
+
+- Unsigned integers: `uint8`, `uint16`, `uint32`, `uint64`
+- Signed integers: `uint8`, `uint16`, `uint32`, `uint64`
+- Floating point numbers: `float16`, `float32`, `float64`
+
+
 ## Components and APIs
 Primary component: `tensor`
 
 Python APIs: [log_tensor](https://ref.rerun.io/docs/python/latest/common/tensors/#rerun.log_tensor),
 
 Rust API: [Tensor](https://docs.rs/rerun/latest/rerun/components/struct.Tensor.html)
+
+## Simple Example
+
+code-example: tensor-simple
```

Comment:
Thinking maybe we should have an example with 1-D tensors as well since they are currently shown as bar-charts